app/src/test/java/pm/gnosis/heimdall/ui/addressbook/AddressBookViewModelTest.kt

```diff
@@ -151,7 +157,7 @@ class AddressBookViewModelTest {
         then(addressBookRepositoryMock).should().addAddressBookEntry(Solidity.Address(BigInteger.ZERO), name, description)
         then(addressBookRepositoryMock).shouldHaveNoMoreInteractions()
         assertEquals(0, testCompletable.callCount)
-        testObserver.assertValue { it is ErrorResult && it.error is AddressBookContract.AddressAlreadyAddedException }
+        testObserver.assertValue { it is ErrorResult && it.error is SimpleLocalizedException }
```

Comment:
You could check the string that is expected here and everywhere where we have the SimpleLocalizedException